src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/tests/AnnexTests.kt

```diff
@@ -19,4 +21,15 @@ class AnnexTests : MontaguTests()
             assertThat(records).isEmpty()
         }
     }
+
+    @Test
+    fun `can read from annex`()
```

Comment:
Might be nice to make this test name more explicit: `can read from annex via foreign table` or similar?